### PR TITLE
docs: Supplementary explanation added

### PR DESCRIPTION
**Pull request by jongmoon**
Ref #14

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,24 @@
...
+
+So we solved following issues that are not resolved in original hammer.js
+
 * [Support for hammer.js in node.js](https://github.com/naver/hammer.js/issues/3)
   - [Make hammerjs server friendly](https://github.com/hammerjs/hammer.js/issues/1060)
   - [Hammer.js fails in node since window is undefined](https://github.com/hammerjs/hammer.js/issues/930)
 * [Support ES Module for Tree Shaking](https://github.com/naver/hammer.js/issues/12)
 * [Change the priority of define and exports for umd](https://github.com/naver/hammer.js/issues/11)
 
+We would maintain this repository to fix bugs remained and enhance it until original repository acivate.
```
Comment (daybrush) [inline README.md]: @jongmoon  acivate? activate?

**Review by daybrush — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1,13 +1,24 @@
 # [hammer.js][hammerjs-url]  [![NPM Version][npm-image]][npm-url]  [![NPM Downloads][downloads-image]][downloads-url]  [![Build Status][travis-image]][travis-url]
 > A JavaScript library for detecting touch gestures.
 
-## Solved Issue
+## What is this repository?
+
+It seems like that `hammer.js` repo is no more acitvated.
```
Comment (daybrush) [inline README.md]: It would be better to write only one of the **repo** and **repository**.

**Review by daybrush — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jongmoon) [reply README.md]: @jongmoon Thank you!

File: README.md (hunk 2 above)
Comment (jongmoon) [reply README.md]: @younkue I fix it to repository.

**Pull request merged**